push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/jms/MessageHolderWithVariantsProducer.java

```diff
@@ -36,21 +34,21 @@
 
     private static final Logger logger = LoggerFactory.getLogger(MessageHolderWithVariantsProducer.class);
 
-    @Resource(mappedName = "java:/queue/APNsPushMessageQueue")
-    private Queue apnsPushMessageQueue;
 
-    @Resource(mappedName = "java:/queue/GCMPushMessageQueue")
-    private Queue gcmPushMessageQueue;
+    private static final String apnsPushMessageQueue = "APNsPushMessageQueue";
+
+
+    private static final String gcmPushMessageQueue = "GCMPushMessageQueue";
+
 
-    @Resource(mappedName = "java:/queue/WNSPushMessageQueue")
-    private Queue wnsPushMessageQueue;
+    private static final String wnsPushMessageQueue = "WNSPushMessageQueue";
```

Comment:
Names for constants should be all UPPERCASE (see [JLS](https://docs.oracle.com/javase/specs/jls/se12/html/jls-6.html#jls-6.5.3) : "_The names of constants in interface types should be, and final variables of class types may conventionally be, a sequence of one or more words, acronyms, or abbreviations, all uppercase, with components separated by underscore "_" characters_")